core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts

```diff
@@ -86,6 +86,13 @@ export class FloatingActionsView {
         this.command = command;
         this.key = key;
         this.tippyOptions = tippyOptions;
+
+        document.addEventListener('DOMContentLoaded', () => {
```

Comment:
This event will be subscribed even if you close the block editor, remove the event listener on `destroy` method in line 196

Reply:
Yeah, we can remove the listener in the destroy method.